openpype/plugins/publish/extract_review.py

```diff
@@ -599,6 +619,10 @@ def _ffmpeg_arguments(
 
         duration_seconds = float(output_frames_len / temp_data["fps"])
 
+        # Define which layer should be used
+        if layer_name:
+            ffmpeg_input_args.extend(["-layer", layer_name])
```

Comment:
Am I correct in understanding that if the input has: `main.r`, `main.g` and `main.b` that here we should be passing it `main`?

Reply:
yes

Reply:
This PR is because of blender, so it is not much surprise I've got this information from blender forum https://blenderartists.org/t/converting-blenders-exr-to-video-with-ffmpeg/1290550/6 .